fe/fe-core/src/test/java/com/starrocks/sql/plan/ExpressionTest.java

```diff
@@ -1249,6 +1249,11 @@ public void testQualifyForWindowFunction() throws Exception {
         plan = getFragmentPlan(sql);
         assertContains(plan, "predicates: 11: dense_rank() > 1");
 
+        // for where
+        sql = "select tc from tall qualify dense_rank() OVER(PARTITION by ta order by tg) > 1 where tb <> 'tb'";
+        plan = getFragmentPlan(sql);
+        assertContains(plan, "predicates: 11: dense_rank() > 1");
```

Comment:
Could you add the `tb <> 'tb'` info in the planFragment to this test? Then we can see the effect of this pr.